spring-cloud-app-broker-core/src/test/java/org/springframework/cloud/appbroker/workflow/instance/AppDeploymentInstanceWorkflowTest.java

```diff
@@ -42,17 +39,39 @@ void setUp() {
 				.build())
 			.build();
 
+		BackingServices backingServices = BackingServices.builder()
+			.backingService(BackingService.builder()
+				.name("service1")
+				.plan("plan1")
+				.build()
+			)
+			.build();
+
 		BrokeredServices brokeredServices = BrokeredServices.builder()
 			.service(BrokeredService.builder()
 				.serviceName("service1")
 				.planName("plan1")
 				.apps(backingApps)
 				.build())
+			.service(BrokeredService.builder()
+				.serviceName("service2_without_backing_app")
+				.planName("plan1")
+				.services(backingServices)
+				.build())
+			.service(BrokeredService.builder()
+				.serviceName("service3_without_backing_app_nor_service")
+				.planName("plan1")
+				.build())
 			.build();
 
 		workflow = new AppDeploymentInstanceWorkflow(brokeredServices);
 	}
 
+	@Test
```

Comment:
This test does not seem very useful, it will always be green. Do you mind deleting it or adding some assertions to demonstrate what it should be asserting? Thanks!

Reply:
Sorry, this was a draft left over of `acceptWithMatchingServiceAndNoBackingApplication` while trying to find right wording for the test case. I'm removing it.

Reply:
Removed in 
45e60b5